src/box/iproto.cc

```diff
@@ -1589,9 +1941,12 @@ static void
 tx_process_call(struct cmsg *m)
 {
 	struct iproto_msg *msg = tx_accept_msg(m);
```

Comment:
Why can't we call `tx_prepare_transaction_for_request` inside `tx_accept_msg`?

Reply:
Fixed